integration-tests/local-integration-tests/src/kusama_test_net.rs

```diff
@@ -1,7 +1,17 @@
 //! Setup of Picasso running as if it is on Kusama relay
 use common::AccountId;
 use cumulus_primitives_core::ParaId;
-use dali_runtime as picasso_runtime;
+
+#[cfg(feature = "dali")]
+use dali_runtime as sibling_runtime;
+#[cfg(feature = "dali")]
```

Comment:
is this correct? shouldn't one of above check for feature != dali?
if it is correct could you please explain more details? @dzmitry-lahoda

Reply:
there will be composable in future.